types.go

```diff
@@ -129,6 +129,14 @@ type Test struct {
 	// Additional properties from XML node attributes.
 	// Some tools use them to store additional information about test location.
 	Properties map[string]string `json:"properties" yaml:"properties"`
+
+	// SystemOut is textual test output for the suite. Usually output that is
```

Comment:
Please tweak the comment, since this is a testcase-level property (opposed to a suite-level one).
```suggestion
	// SystemOut is textual output for the test case. Usually output that is
```

Reply:
Right. Missed that.